src/ctia/entity/relationship.clj

```diff
@@ -145,8 +144,7 @@
                                    source-short-id
                                    identity-map
                                    {}))
-              target-ref (short-id->long-id id
-                                            get-http-show)]
+              target-ref (short-id->long-id id)]
```

Comment:
Moved from `ctim.domain.id/short-id->long-id` to `ctia.domain.entities/short-id->long-id`. Helps future refactoring.